client/blocks/image-editor/index.jsx

```diff
@@ -225,6 +223,7 @@ const ImageEditor = React.createClass( {
 			<div className={ classes }>
 				{ this.state.canvasError && this.renderError() }
 
+				<CloseOnEscape onEscape={ this.onCancel } />
```

Comment:
Hi @retrofox!
Just to be sure, do you mean changing the name of the `onCancel` method itself?
It definitely could be more descriptive!
Happy to do that - I'll add a commit soon :)